_source/logzio_collections/_security-sources/cynet.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Ship logs from Cynet
+logo:
+  logofile: cynet.png
+  orientation: vertical
+data-source: Cynet
+templates: ["network-device-filebeat"]
+contributors:
+  - nshishkin
+shipping-tags:
+  - firewalls
+order: 1380
+---
+[Cynet](https://www.cynet.com/platform/) is a cybersecurity asset management platform. This topic describes how to send system logs from your Cynet platform to Logz.io.
+
+**Before you begin, you'll need**: 
+
+* An active Cynet license
+* Cynet login credentials 
+* An active account with Logz.io
+* Filebeat 7 installed on your machine
+* Root priveleges on your machines 
+
+<div class="tasklist">
+
+
+##### Configure Cynet to send syslog notifications to a remote Syslog server running Filebeat
+
+1. On your Cynet web interface, go to **Setting > Advanced**.
+![Advanced options](https://dytvr9ot2sszz.cloudfront.net/logz-docs/Cynet/cynet1.png)
+2. Select the box beside **Send Audit Records to SIEM**.
+3. Go to **Configuration > SIEM settings** and enable the following configuration:
+![SIEM settings](https://dytvr9ot2sszz.cloudfront.net/logz-docs/Cynet/cynet2.png)
+   * **IP** - IP
+   * **Port** - 514
+   * **IP address** - `<ADDRESS-OF-YOUR-FILEBEAT-SERVER>>`, e.g. 100.25.154.13:9000
+
+<!-- info-box-start:info -->
+By default, syslog will be forwarded over port 514. Feel free to adjust this, based on your preference or availability, but be sure to note any change to this port in the Filebeat configuration.
+{:.info-box.note}
+<!-- info-box-end -->
+
+{% include log-shipping/certificate.md %}
+
+
+##### Configure Filebeat
```

Comment:
@nico-shishkin 
almost every security product that working with syslog integration has the same config
Lets create a snippet for it so it will be really easy to create new pages